corehq/apps/reports/static/reports/js/filters/case_list_explorer.js

```diff
@@ -1,24 +1,24 @@
-var suggestedCaseTypes = function(){
-    // Adds the required properties to filter the case type autocomplete dropdowns
-    var self = this;
-    self.currentCaseType = ko.observable('');
-    $('#report_filter_case_type').on('change', function(e){
-        self.currentCaseType(e.val);
-    });
-
-    self.suggestedProperties = ko.computed(function(){
-        if (self.currentCaseType() === ''){
-            return self.allCaseProperties;
-        }
-        return _.filter(self.allCaseProperties, function(prop){
-            return prop['case_type'] === self.currentCaseType() || prop['case_type'] === null;
-        });
-    });
-};
-
 hqDefine("reports/js/filters/case_list_explorer", ['jQuery', 'underscore', 'knockout'], function($, _, ko) {
     'use strict';
 
+    var suggestions = function(){
```

Comment:
Picky, but would you consider renaming this? I squinted at it for a few minutes thinking it was meant to be a class (despite the comment), and was about to complain about the inheritance setup. `applySuggestions`?

Reply:
:+1: